### DISCUSSION TITLE
【翻訳案】The Bionic Booster Pack 初出の単語

### DISCUSSION THREAD

**Discussion by nsk4762jp** [nsk4762jp/OxygenNotIncluded-Japanese#406] [Ideas]
DLC初出の単語について、この場でいろいろ検討できたらなと思います。
まだ一覧化できてませんが、随時リストアップしていく予定です。

### 検討中

原語 | 翻訳案 | 備考
----- | ----- | -----

### 確定

原語 | 翻訳案 | 備考
----- | ----- | -----
Bionic | バイオニック  | 通常は生体工学を指す  
Bionic Duplicant | バイオニック複製人間  | 製造方法が通常の複製人間と異なる、有機体と無機物の混合物 
Bionic Parts | バイオニックパーツ | バイオニック複製人間の無機物部分  
Gunk | 残滓  | Bionic Duplicant が排出する。複製人間のおしっこ(汚染水)相当のもの 
Solid Gunk | 固体残滓  | Gunk が凍結して固体になったもの
Gunk Extractor | 残滓排出装置 | Gunk を排出するための設備  
Masterworks Art Booster | 傑作ブースター | Masterworks(傑作)スキルに相当
Strength Booster | 筋力ブースター |  
Construction Booster | 建築ブースター |  
Grilling Booster | グリル調理ブースター | Grilling II(グリル調理 II)スキルに相当
Digging Booster | 採掘ブースター |  
Extreme Digging Booster | 超硬質採掘ブースター | 超硬質採掘&超々硬質採掘&危険物採掘スキルに相当
Crop Tending Booster | 作物の手入れブースター |  
Advanced Medical Booster | 高度な医療看護ブースター | Advanced Medical Care(高度な医療看護)スキルに相当
Electrical Engineering Booster | 電気工学ブースター |  
Mechatronics Engineering Booster | メカトロニクス工学ブースター |  
Piloting Booster | 操縦ブースター | Spaced Out! DLC がある場合のみ
Rocketry Booster | ロケット工学ブースター | Spaced Out! DLC がない場合のみ
Ranching Booster | 牧畜ブースター |  
Researching Booster | 研究ブースター |  
Astronomy Booster | 天文学ブースター |  
Applied Sciences Booster | 材料工学ブースター |  
Suit Training Booster | スーツトレーニングブースター |  
Tidying Booster | 清掃ブースター
Gear Balm | ギア軟膏 | バイオニック複製人間に潤滑油を補給する道具。タグ名が LUBRICATIONSTICK なので、リップクリームのように塗り塗りして使うもの？
Gear Oil | ギアオイル | 個別のアイテムではなく、元素の分類名（原油、植物油が該当）バイオニック複製人間に注油する潤滑剤全般のこと
Power Bank | 電力バンク | ポータブルバッテリー。  
Metal Power Bank | メタル電力バンク | 金属鉱石を原料とする電力バンク
Atomic Power Bank | 原子力電力バンク | `濃縮ウラン` を原料とするセルフ充電式電力バンク
Uranium Ore Power Bank | ウラン鉱石電力バンク | `ウラン鉱石` を原料とする電力バンク
Eco Power Bank | エコ電力バンク | 再充電可能な電力バンク
Large Discharger | 大型放電器 | 電力バンクを繋げて設備の電力源とする
Compact Discharger | コンパクト放電器 | 電力バンクを繋げて設備の電力源とする 
Bionic service station | バイオニック整備端末 | 設備の分類名？バイオニック複製人間が使う設備っぽい
Lubrication Station | 注油端末 | 残滓処理装置と同じく、バイオニック複製人間が使うWorkshop内の設備
Flydo | 飛行ドローン | リモートワーカー   
Remote Controller | リモート コントローラー | 複製人間がリモートワーカーを操作するための設備
Remote Worker | リモートワーカー | 複製人間が遠隔操作できる人型ロボット
Remote Worker Dock | リモートワーカー ドック | リモートワーカーの待機場所
Heavy-duty generator | 高負荷発電機 | 設備の分類名？Heavy-duty（高耐久）という名前だが、単に発電量が多いだけっぽい。
Basic generator | 簡易発電機 | 設備の分類名？マイクロチップの電力調整ができないっぽい。

**Comment by denev-rove** [comment]
`Basic generator` は`人力発電機`、`Heavy-duty generator` はそれ以外の発電機の分類でした。
マイクロチップで出力調整できるかどうかの区別みたいです。

![大型発電機](https://github.com/user-attachments/assets/6f6f5195-4675-4126-89d8-47f6bfe14d9a)

**Reply by tetuis** [reply]
今後サイズは大きくても出力調整ができない発電機が出る可能性も考えられるため、元々の意味から離れた大型/小型という分類名は避けた方がいいのではないかと思います。

**Reply by nsk4762jp** [reply]
そうですねえ。
`Heavy-duty` には `高負荷` という意味もあるようですし、`Basic` は `Simple(簡易)` という意味もあるようなので、

- Heavy-duty Generator: 高負荷発電機
- Basic Generator: 簡易発電機

といった感じでしょうか。

**Comment by denev-rove** [comment]
翻訳案の提案です。
- `Power Bank`
ゲーム中の機能は蓄電池そのものなので、`Power` は普通に `電力` という意味でいいと思います。用語を揃えて`電力バンク`などとしたいですが、いかがでしょうか。（あるいは単に`電池`とか）

- `Gunk Extractor`
Extract: 抜き出す
「処理装置」だと（廃棄物処理施設のように）残滓自体をどうにかするような感じを受けるので、`残滓排泄装置`というのはどうでしょうか。バイオニック複製人間にとってのトイレであることが分かりやすくなると思います。（`排泄`が直接的すぎるなら`排出装置`とか）

**Reply by nsk4762jp** [reply]
そうですねえ。既存の `データバンク` に倣って `パワーバンク` としただけなので、`電力バンク` でいいかもしれませんね。
`Gunk Extractor` については `残滓排出装置` にしましょうか。

**Reply by denev-rove** [reply]
後出しですみません、`Power Bank` は単に `電池` でもいいかもしれないと思いました。`エコ電池` `核電池`（`原子力電池`）など

`残滓排出装置`、いいと思います。`残滓`の排出は原語でもいろいろな動詞が使われていて(expel, extract)訳に迷いましたが、ひとまず「排出」で統一して作業します。

**Reply by nsk4762jp** [reply]
電池は英語で `Battery` なので既存のバッテリーとの混同を避けたいところです。そういうところも含めて原語はわざと `Portable/Mobile Battery` ではなく `Power Bank` にしているのかもしれませんね。

**Reply by denev-rove** [reply]
確かにそうですね。では`電力バンク`で作業します。

**Comment by denev-rove** [comment]
まだ一覧にない文言を気付いた範囲で列挙します。
括弧内の私案も一考していただければと思います。（訳せるものはなるべく日本語に訳す方針）
|原語|翻訳案<br>（私案）|備考|
|-|-|-|
|Gear Balm|ギア軟膏|バイオニック複製人間に潤滑油を補給する道具。<br>タグ名が `LUBRICATIONSTICK` なので、リップクリームのように塗り塗りして使うもの？|
|Gear Oil|ギアオイル|個別のアイテムではなく、元素の分類名（`原油`、`植物油`が該当）<br>バイオニック複製人間に注油する潤滑剤全般のこと|
|Remote Controller|リモート コントローラー<br>（遠隔作業端末）|複製人間がリモートワーカーを操作するための設備|
|Remote Worker|リモートワーカー<br>（遠隔作業ロボ）|複製人間が遠隔操作できる人型ロボット|
|Remote Worker Dock|リモートワーカー ドック<br>（遠隔作業ロボ ドック）|リモートワーカーの待機場所|

**Reply by nsk4762jp** [reply]
ギア関連は特に異存はありません。
リモート系はリモートワーク(テレワーク)という言葉がコロナ禍で浸透したので敢えて訳す必要も無いかなと思っています(UI の都合上どうしても厳しい場合は除く)。

**Reply by denev-rove** [reply]
UIで気になるのは設備設置パネルですが、どうしても厳しいという程でもないです。

![remote2](https://github.com/user-attachments/assets/2138b3b5-1d9c-41f1-ae74-a9dae9b344b2) ![remote](https://github.com/user-attachments/assets/e769af51-91fa-4928-aa74-4461beda631e)

**Reply by nsk4762jp** [reply]
リモートコントローラーってツインスティックでガコガコやるタイプだと思ったので、このグラフィックだとVRの全身トレースみたいですね。
今のところ `xxx Station` を `xxx 端末` と翻訳しているので、`Remote Controller` はそのまま `リモートコントローラー` にしましょう。

**Comment by denev-rove** [comment]
バイオニック複製人間が使うブースターの一覧です。
ゲーム内での見た目はファミコンのカセットのような形状で、バイオニック複製人間が装着するときは口から飲み込むような動作をします。（スロットは体内にあるっぽい）

文章では、バイオニック複製人間の空きスロットに `install` するものとして説明されていますが、この動詞の訳を`装着する`等で統一したいです。（インストールだとソフトウェアの印象が強い）

|原語|翻訳案<br>（私案）|備考|
|-|-|-|
|Masterworks Art Booster|傑作ブースター|`Masterworks`(`傑作`)スキルに相当|
|Strength Booster|筋力ブースター||
|Construction Booster|建築ブースター||
|Grilling Booster|グリルブースター<br>（グリル調理ブースター）|`Grilling II`(`グリル調理 II`)スキルに相当|
|Digging Booster|採掘ブースター||
|Extreme Digging Booster|超硬質採掘ブースター|`超硬質採掘`&`超々硬質採掘`&`危険物採掘`スキルに相当|
|Crop Tending Booster|作物の手入れブースター||
|Advanced Medical Booster|高度な医療看護ブースター<br>（先進医療ブースター）|`Advanced Medical Care`(`高度な医療看護`)スキルに相当|
|Electrical Engineering Booster|電気工学ブースター||
|Mechatronics Engineering Booster|メカトロニクス工学ブースター||
|Piloting Booster|操縦ブースター|Spaced Out! DLC がある場合のみ|
|Rocketry Booster|ロケット工学ブースター|Spaced Out! DLC がない場合のみ|
|Ranching Booster|牧畜ブースター||
|Researching Booster|研究ブースター||
|Astronomy Booster|天文学ブースター||
|Applied Sciences Booster|材料工学ブースター||
|Suit Training Booster|スーツトレーニングブースター<br>（スーツ訓練ブースター）||
|Tidying Booster|清掃ブースター||

- `Grilling Booster`
スキル名に合わせて`グリル調理ブースター`ではどうでしょうか。料理のスキルであることが伝わりやすくなると思います。
- `Advanced Medical Booster`
原語はスキル名とわずかに異なるので、そのまま訳すと`先進医療ブースター`になると思います。個人的にはこちらのほうがすっきりして好きですが、スキル名と一致しないことが気になるなら取り下げます。
- `Suit Training Booster`
名前が長いので、簡素に`スーツ訓練ブースター`にしたいです。（できればこの機にスキル名の方も`EXOスーツ訓練`とかにしたい）

よろしくお願いします。

**Reply by nsk4762jp** [reply]
ブースターの名称は上がる能力の名前(かスキル名)がそのまま付いてたほうが分かりやすくていいと思いますね。既存のスキル名は変えるとまた文句が出そうなので据え置きにしましょう。

**Reply by denev-rove** [reply]
うーん、残念。
私の場合、スキルは画面上の位置で把握していて、スキル名はあまり意識してこなかったので影響は少ないかなと思ったのですが、分かりました。

今回はじめて原語を確認して、`Suit Sustainability Training` の `Sustainability` の部分を上手く言い表せないかなと色々考えたのですが、難しいですね。

**Reply by denev-rove** [reply]
`Grilling Booster` はスキル名をそのまま付けるなら`グリル調理ブースター`になると思いますが、訳語の統一（[a61a264](https://github.com/nsk4762jp/OxygenNotIncluded-Japanese/pull/421/commits/a61a2646cde558177c80a4892c01c678d55cfc65)）で`グリルブースター`に統一されているようです。

**Comment by denev-rove** [comment]
翻訳の提案ではないですが、一覧にある
- `Squash Power Bank`
- `Shine Egg Power Bank`
- `Muckroot Power Bank`
- `Slug Egg Power Bank`
- `Sucrose Power Bank`

これらの`電力バンク`は現バージョンでは没になってるみたいです。（翻訳ファイルに存在しない）
作業に直接かかわりはありませんが、一覧としては別立てにしたほうが混乱がないのかなと思います。
